net/netclient/patches/010-install-initd-fix.patch

```diff
@@ -0,0 +1,59 @@
+--- a/daemon/initd_linux.go
++++ b/daemon/initd_linux.go
+@@ -1,37 +1,12 @@
+ package daemon
+ 
+ import (
+-	"errors"
+-	"os"
+-
```

Comment:
Again... why?

Reply:
The compiler fails if these lines are not removed when the other lines from this patch are removed.

Reply:
No, you're wrong.